### PR TITLE
noetic magni_bringup README documentation

### PR DESCRIPTION
**Pull request by JanezCim**
Can you take a look at the readme and comment on what else needs to be added?

### CODE REVIEW

File: magni_bringup/README.md [hunk 1]
```diff
@@ -1,40 +1,48 @@
...
-3.) `launch_core.py` creates and launches `core.launch` (by default in `/tmp/core.launch`) which starts up the rest of the robot nodes that are variable (change with robot config):
-  a) motor node: runs ubiquity_motor node with the parameters extracted from either `/etc/ubiquity/robot.yaml` OR `magni_bringup/param/robot.yaml` with respective priorities
-  b) robot description: all robot static and dynamic transforms. It gets its camera and lidar extrinsics from either `~/.ros/extrinsics/<SENSOR>_extrinsics_<POSITION>.yaml` OR `magni_description/extrinsics/<SENSOR>_extrinsics_<POSITION>.yaml` with respective priorities. 
-  c) other nodes like `controller_spawner`, `diagnostic_aggregator`, `oled_display_node`,... Exactly what gets launched can be seen in `magni_bringup/param/core_launch.em` from which core.launch is generated. Any addition to core.launch should be added inside the `core_launch.em` file
 
-**Useful commands for debugging**
-If new additions need to be added to core.launch, add them into `magni_bringup/param/core_launch.em`, then edit the launch_core.py accordingly. You can then run the commands:
+`systemctl start magni-base.service`:
+  - roscore
```
Comment (rohbotics) [inline magni_bringup/README.md]: roscore is a separate systemd service roscore.service

File: magni_bringup/README.md [hunk 2]
```diff
@@ -1,40 +1,48 @@
...
+  - env variables
```
Comment (rohbotics) [inline magni_bringup/README.md:7]: ROS_HOSTNAME and ROS_MASTER_URI are found in `/etc/ubiquity/env.sh`

**Review by rohbotics — CHANGES_REQUESTED** [review]
Some minor things I noticed. Other things look good.

File: magni_bringup/README.md (hunk 1 above)
Comment (JanezCim) [reply magni_bringup/README.md]: fixed

**Review by rohbotics — APPROVED** [review]

**Pull request merged**